lib/vizkit/cplusplus_extensions/virtual_joystick.rb

```diff
@@ -4,7 +4,7 @@
     end
 
     widget.connect(SIGNAL('axisChanged(double,double)')) do |x, y|
-        value = Types::Base::MotionCommand2D.new
+        value = Types::Base::Commands::Motion2D.new
```

Comment:
Would probably be better to move to the new typelib syntax while you're at it `Types.base.commands.Motion2D.new`